capstone/src/main/resources/static/map.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+  
+  <head>
+    <meta charset="UTF-8">
+    <title> Map </title>
+    <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyA59BNac2K3n6PHyjjF5dxkZ9AbBHIxHSM"></script>
+    <script src="script.js"> </script>
+    <link rel="stylesheet" href="style.css">
+  </head>
+
+  <body onload="createMap()">
+
+    <div class="menu">
+      <nav>
+        <a href="event.html"> Create Event </a>
+      </nav>
+    </div> 
+
+    <h1> Map of New York </h1>
+    <div id="map"> </div>
+  </body
```

Comment:
closing angle bracket is missing.
```suggestion
  </body>
```